src/features/views/hooks/useItemsMutations.ts

```diff
@@ -0,0 +1,54 @@
+import { useAppSelector } from 'core/hooks';
+import useFolder from './useFolder';
+import useViewMutations from './useViewMutations';
+
+interface UseItemsMutationsReturn {
+  itemIsRenaming: (type: 'folder' | 'view', id: number) => boolean;
+  moveItem: (
+    type: 'folder' | 'view',
+    id: number,
+    newParentId: number | null
+  ) => void;
+  renameItem: (type: 'folder' | 'view', id: number, title: string) => void;
+}
+
+export default function useItemsMutations(
```

Comment:
The same naming problem applies here, the `Items` term is very generic. I would rename to something like `useViewBrowserMutations()`.

Or I would merge this hook with the `useItems` hook and call it `useViewBrowser()`.